release/_publishing.sh

```diff
@@ -337,49 +353,60 @@ function _update_bundles_yml {
-		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\" = {}" "${_BASE_DIR}/bundles.yml"
-		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${product_version_key}.nightly\".bundle_url = \"${nightly_bundle_url}\"" "${_BASE_DIR}/bundles.yml"
+		yq --indent 4 --inplace eval "del(.\"${product_version_key}\".\"${product_version_key}.nightly\")" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
+		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\" = {}" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
+		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${product_version_key}.nightly\".bundle_url = \"${nightly_bundle_url}\"" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 	fi
 
 	if [[ "${_PRODUCT_VERSION}" == 7.4.*-ga* ]]
 	then
 		local ga_bundle_url="releases-cdn.liferay.com/portal/${_PRODUCT_VERSION}/"$(curl -fsSL "https://releases-cdn.liferay.com/portal/${_PRODUCT_VERSION}/.lfrrelease-tomcat-bundle")
 
-		perl -i -0777pe 's/\s+latest: true(?!7.4.13:)//' "${_BASE_DIR}/bundles.yml"
+		perl -i -0777pe 's/\s+latest: true(?!7.4.13:)//' "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 
-		sed -i "/7.4.13:/i ${product_version_key}:" "${_BASE_DIR}/bundles.yml"
+		sed -i "/7.4.13:/i ${product_version_key}:" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 
-		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\".bundle_url = \"${ga_bundle_url}\"" "${_BASE_DIR}/bundles.yml"
-		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\".latest = true" "${_BASE_DIR}/bundles.yml"
+		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\".bundle_url = \"${ga_bundle_url}\"" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
+		yq --indent 4 --inplace eval ".\"${product_version_key}\".\"${_PRODUCT_VERSION}\".latest = true" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 	fi
 
-	sed -i "s/[[:space:]]{}//g" "${_BASE_DIR}/bundles.yml"
+	sed -i "s/[[:space:]]{}//g" "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 
-	truncate -s -1 "${_BASE_DIR}/bundles.yml"
+	truncate -s -1 "${_PROJECTS_DIR}/liferay-docker/bundles.yml"
 
 	if [ -z "${LIFERAY_RELEASE_TEST_MODE}" ]
 	then
-		git add "${_BASE_DIR}/bundles.yml"
-
-		git commit -m "Add ${_PRODUCT_VERSION} to bundles.yml."
+		commit_to_branch_and_send_pull_request \
+			"${_PROJECTS_DIR}/liferay-docker/bundles.yml" \
+			"Add ${_PRODUCT_VERSION} to bundles.yml." \
+			"update-bundles-yml-branch" \
+			"master" \
+			"brianchandotcom/liferay-docker" \
+			"Add ${_PRODUCT_VERSION} to bundles.yml."
+
+		if [ "${?}" -eq "${LIFERAY_COMMON_EXIT_CODE_BAD}" ]
+		then
+			lc_log ERROR "Unable to send to brianchandotcom/liferay-docker."
 
-		git push upstream master
+			return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+		else
+			lc_log INFO "The bundles.yml was updated successfully."
```

Comment:
@kiwm, please update the message informing that the pull request was sent successfully instead.